actionpack/test/dispatch/host_authorization_test.rb

```diff
@@ -451,7 +451,7 @@ class HostAuthorizationTest < ActionDispatch::IntegrationTest
     }
 
     assert_response :forbidden
-    assert_match "Blocked host: attacker.com#x.example.com", response.body
```

Comment:
Interesting this behavior changed, I wonder if we can keep it?

Reply:
Yes. I changed it back. It actually makes no difference, for some strange reasons.

Reply:
You mean the test passed either way? My guess is because they both will match the regex.